assets/cross-sell-carousel.css

```diff
@@ -0,0 +1,88 @@
+@media (min-width: 751), (max-width: 2000px) {
+
+  .cross-sell-product-title {
+    /* show only 2 lines of text */
+    display: block;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
+    overflow: hidden;
+    max-height: 3.6em;
+    line-height: 1.8em;
+    /* end */
+    padding: 0 10px;
+    font-size: 12px;
+  }
+
+  .glide__arrow--left {
+    left: -1em;
+  }
+
+  .glide__arrow--right {
+    right: -1em;
+  }
+}
+
+@media (max-width: 750px) {
+
+  .cross-sell-product-title {
+    height: 60px;
+  }
+
+  .glide__arrow--left {
+    left: 1em;
+  }
+
+  .glide__arrow--right {
+    right: 1em;
+  }
+}
+
+@media (max-width: 450px) {
+
+  .cross-sell-product-title {
+    /* show only 2 lines of text */
+    display: block;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
+    overflow: hidden;
+    max-height: 3.6em;
+    line-height: 1.8em;
+    /* end */
+    padding: 0 10px;
+  }
+
+  .glide__arrow--left {
+    left: -0.3em;
+  }
+
+  .glide__arrow--right {
+    right: -0.3em;
+  }
+}
+
+@media (min-width: 750), (max-width: 850px) {
+
+  .button,
+  .shopify-challenge__button,
+  .customer button,
+  button.shopify-payment-button__button--unbranded {
+    min-width: 110px;
+    min-height: 45px;
+  }
+}
+
+@media (min-width: 990), (max-width: 1100px) {
```

Comment:
Carefull with this, it can easily become confusing.

Reply:
Yeah, this is still an issue, you should remove the maxwidth statements, it over complicates the media queries.
Since you're doing mobile first, you can use min width.